elastic/kibana
# [Guided onboarding] [Security] Add EUI tour to Integrations pages

Fixes https://github.com/elastic/kibana/issues/139712

### Summary

This PR is a part of the [guided onboarding](https://github.com/elastic/mx-team/issues/93) experience project and aims to add a Eui tour to the Elastic Defend integration card.


### How to test
<details>

1. Start es. `yarn es snapshot --license trial`
2. Start Kibana. `yarn start --run-examples`
3. In Kibana, navigate to `http://localhost:5601/app/guidedOnboardingExample` and start the Security guide. Make sure the green guided onboarding button is displayed in the header and the first step is active.
4. In the guided onboarding checklist panel, click the "Start" button in the first step. Or alternatively navigate to `http://localhost:5601/app/integrations/browse/security`
5. The tour should be displayed on the Elastic Defend card.

</details>


### Screenshot
<details>
<img width="1158" alt="Screen Shot 2022-10-28 at 11 23 32" src="https://user-images.githubusercontent.com/53621505/198517746-3253d035-99f0-4b6f-8f3b-2cf914a8f6d0.png">

</details>
